clients/admin-ui/src/features/integrations/AddIntegrationModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Flex, Spacer, UseDisclosureReturn } from "fidesui";
+import { useState } from "react";
+
+import AddModal from "~/features/configure-consent/AddModal";
+import BigQueryOverview from "~/features/integrations/bigqueryOverviewCopy";
+import ConfigureIntegrationForm from "~/features/integrations/ConfigureIntegrationForm";
+import IntegrationBox from "~/features/integrations/IntegrationBox";
+import { AccessLevel, ConnectionType } from "~/types/api";
+
+const BQ_PLACEHOLDER = {
+  name: "Google BigQuery",
+  key: "bq_placeholder",
+  connection_type: ConnectionType.BIGQUERY,
+  access: AccessLevel.READ,
+  created_at: "",
+};
+
+const AddIntegrationModal = ({
+  isOpen,
+  onClose,
+}: Pick<UseDisclosureReturn, "isOpen" | "onClose">) => {
+  const [step, setStep] = useState(0);
+
+  const onCancel = () => {
+    setStep(0);
+    onClose();
+  };
+
+  return (
+    <AddModal
+      isOpen={isOpen}
+      onClose={onClose}
+      title="Google BigQuery integration"
+    >
+      {step === 0 && (
+        <>
+          <IntegrationBox
+            integration={BQ_PLACEHOLDER}
+            button={
```

Comment:
Is there a need to pass the button to the IntegrationBox component? Is it going to look very different depending on the Integration?  
If not, I think it would be better if the button was part of the IntegrationBox component. The IntegrationBox component could then have a "onAccept" or "onConfigure" prop so we can do the "setStep(1)" here.